README.rdoc

```diff
@@ -57,6 +57,25 @@ Example to read/write files in Google Drive:
 
   # Updates content of the remote file.
   file.update_from_file("/path/to/hello.txt")
+  
+Example to read/write file in Google Drive with token that was stored in a local file (just request access authorization once)
```

Comment:
I believe putting all of these examples in README makes README too long. How about replacing the example code above with yours (with saved_session)? But please clarify that saved_session prompts the credential via command line, because it may not always be what people want.

The way to use refresh token is documented here:
http://gimite.net/doc/google-drive-ruby/GoogleDrive.html#method-c-login_with_oauth
So how about just putting a link to the page?